cmd/juju/charmhub/info.go

```diff
@@ -91,6 +94,14 @@ func (c *infoCommand) Init(args []string) error {
 		return errors.Trace(err)
 	}
 	c.charmOrBundle = args[0]
+
+	switch c.unicode {
+	case "auto", "never", "always":
+	case "":
```

Comment:
how is this possible if the default is auto?  `juju info mysql --unicode "" ` is an error in my mind.